sdk/authorization/Azure.ResourceManager.Authorization/src/Generated/Models/AuthorizationProviderOperationInfo.Serialization.cs

```diff
@@ -49,22 +49,15 @@ void IJsonModel<AuthorizationProviderOperationInfo>.Write(Utf8JsonWriter writer,
             }
             if (Optional.IsDefined(Properties))
             {
-                if (Properties != null)
-                {
-                    writer.WritePropertyName("properties"u8);
+                writer.WritePropertyName("properties"u8);
 #if NET6_0_OR_GREATER
 				writer.WriteRawValue(Properties);
 #else
-                    using (JsonDocument document = JsonDocument.Parse(Properties))
-                    {
-                        JsonSerializer.Serialize(writer, document.RootElement);
-                    }
-#endif
-                }
-                else
+                using (JsonDocument document = JsonDocument.Parse(Properties))
                 {
-                    writer.WriteNull("properties");
+                    JsonSerializer.Serialize(writer, document.RootElement);
                 }
+#endif
             }
```

Comment:
this change should be fine, because the else branch will never execute.
`Optional.IsDefined(Properties)` is equivalent to `Properties != null` becasue the type of `Properties` here is `BinaryData` which is a reference type.